handycon.py

```diff
@@ -662,6 +668,45 @@ async def capture_keyboard_events():
                             elif active == [] and seed_event.code == 125 and button_on == 0 and  event_queue == [] and shutdown == True:
                                 shutdown = False
 
+                        case "AOK":
+                            # BUTTON 1 (Turbo button can't used) 
+
+                            # BUTTON 2 (Default: QAM) Long press yellow
+                            if active == [34, 125] and button_on == 1 and button2 not in event_queue:
+                                event_queue.append(button2)
+                            elif active == [] and seed_event.code in [34, 125] and button_on == 0 and button2 in event_queue:
+                                this_button = button2
+
+                            # BUTTON 3 (Default: Toggle Gyro) Short press yellow + keyboard
+                            if active == [24, 97, 125] and button_on == 1 and button3 not in event_queue and gyro_device:
```

Comment:
On the OXP Mini when you press yellow + KB I believe you get values [97, 100, 111] and I see here that you are catching the signal for KB [24, 97, 125]. It would seem that if a gyro was detected you are sending both button 3 and button 4 (ESC and KB).

Is there any unexpected behavior when opening/closing various menus?

Reply:
> A report from another owner of the Aokzoe mentioned that the menus do not open/close consistently for them on their device. Does yours work differently?

This is their official firmware issue
I have contacted them
They haven't replied yet if they will fix it with a new update
I will follow up on this issue

Reply:
> On the OXP Mini when you press yellow + KB I believe you get values [97, 100, 111] and I see here that you are catching the signal for KB [24, 97, 125]. It would seem that if a gyro was detected you are sending both button 3 and button 4 (ESC and KB).
> 
> Is there any unexpected behavior when opening/closing various menus?

You wait till I test to see if there's an error

Reply:
You're right. I need to change this code